web/src/pages/aqua_sign_wokflow/ContractDocument/components/signature_overlay.tsx

```
@@ -1,162 +1,155 @@
+                pointerEvents: 'auto',
+                cursor: signature.isDragging ? 'grabbing' : 'grab',
+                zIndex: signature.isDragging ? 20 : 10,
+                border: signature.isDragging ? '2px dashed blue' : 'none',
+                maxWidth: '250px',
+                padding: '4px',
+                borderRadius: '4px',
+                borderColor: 'rgb(216, 216, 216)',
+            }}
+            onMouseDown={e => handleDragStart?.(e, signature.id)}
+            onTouchStart={e => handleDragStart?.(e, signature.id)}
+        >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)